connection/connection.go

```diff
@@ -0,0 +1,21 @@
+package connection
+
+// ID uniquely identifies a conneciton.
+type ID string
+
+// Connection is a configuration used by the connector to transport data from source to the EG (target).
+type Connection struct {
+	Space  string           `json:"space"`
+	ID     ID               `json:"connectionId"`
+	Target string           `json:"target"`
+	Source AWSKinesisSource `json:"source"`
+}
+
+// AWSKinesisSource is a configuration used to configure AWS Kinesis stream as a source.
+type AWSKinesisSource struct {
+	StreamName         string `json:"streamName"`
+	Region             string `json:"region"`
+	AWSAccessKeyID     string `json:"awsAccessKeyId,omitempty"`
```

Comment:
is the `awsAccessKeyId` a defautl from AWS? If not, I'd prefer to mark anything with ID to use uppercase, like `awsAccessKeyID`

Reply:
I just copied that from Event Gateway code. In EG core we use that convention. I might not be perfect but at least it's consistent.